docs/platform/howto/create-billing-groups.rst

```diff
@@ -0,0 +1,36 @@
+Create billing groups
+=====================
+
+With billing groups you can set up billing profiles to be used across multiple projects. A consolidated invoice is created for each billing group. To learn more about the key benefits of billing groups read the :doc:`related documentation <../concepts/billing-groups>`.
```

Comment:
I'd be good to add somewhere here what the user can learn from this article, although it may seems obvious.